cmd/vulndash/cmd/root.go

```diff
@@ -102,6 +104,13 @@ func init() {
 		"info",
 		"the logging verbosity, either 'panic', 'fatal', 'error', 'warn', 'warning', 'info', 'debug' or 'trace'",
 	)
+
+	rootCmd.PersistentFlags().Int32Var(
+		&opts.pageSize,
+		pageSizeFlag,
+		200,
```

Comment:
As a follow-up, let's set this as a default (`DefaultPageSize`) in `pkg/vulndash/adapter/adapter.go`.

Reply:
https://github.com/kubernetes/release/pull/1668